src/components/SecondWorkspace.js

```diff
@@ -0,0 +1,48 @@
+import { Flex, Image } from "@chakra-ui/react";
+import { Chair } from "./Chair";
+
+export const SecondWorkspace = () => {
+  return (
+    <Flex
+      direction="row"
+      w="40%"
+      h="60vh"
+      align="center"
+      justify="space-evenly"
+    >
+      <Flex
+        direction="column"
+        w="100%"
+        h="60vh"
+        align="center"
+        justify="space-evenly"
+      >
+        <Chair orientation="left" />
+        <Chair orientation="left" />
+        <Chair orientation="left" />
+        <Chair orientation="left" />
```

Comment:
This could be abstracted more, e.g. we can have a method that takes the number of chairs  and orientation e.g. `setupChairs(numberOfCharis=4, orientation="left")`